komodo/build.py

```diff
@@ -248,8 +248,6 @@ def resolve(input_str):
             msg = "pypi_package_name is only valid when building with pip"
             raise ValueError(msg)
 
-        package_name = current.get("pypi_package_name", package_name)
```

Comment:
Have you tested this? I just worry that this will affect the other build types somehow.

Reply:
Testing in Azure:  
https://github.com/equinor/komodo-releases/actions/runs/13542199588

Reply:
https://github.com/equinor/komodo-releases/actions/runs/13542599931/job/37846905966